src/Map/Tile.cpp

```diff
@@ -125,9 +125,14 @@ Robot* Tile::robot()
 /**
  * 
  */
-float Tile::distanceTo(Tile* t)
+float Tile::distanceTo(Tile* tile)
 {
-	int x = t->x() - Tile::x();
-	int y = t->y() - Tile::y();
-	return static_cast<float>(sqrt((x * x) + (y * y)));
+	return distanceTo(tile->position());
+}
+
+
+float Tile::distanceTo(NAS2D::Point<int> point)
```

Comment:
The argument name and type is unclear whether or not this is tile space (i.e. tile coords), or world space (pixel-perfect, sub-tile coordinates).

Additionally, are you counting from the corner of a tile or the center?

Reply:
The name isn't really the concern of this PR. It used an existing name. This should be addressed as it's own issue.

Reply:
And to add to that, it's on the `Tile` class, so that should give a hint.